### PR TITLE
cdc: add warn for event filter

### PR DESCRIPTION
**Pull request by wk989898**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-changefeed-config.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ rules = ['*.*', '!test.*']
 # 第二个事件过滤器规则
 # [[filter.event-filters]]
 # matcher = ["test.fruit"] # 该事件过滤器只应用于 test.fruit 表
-# ignore-event = ["drop table", "delete"] # 忽略 drop table 的 DDL 事件和 delete 类型的 DML 事件
+# ignore-event = ["drop table", "delete"] # 忽略 drop table 的 DDL 事件和 delete 类型的 DML 事件，注意 TiDB 在更新聚簇索引的列值时，会将一个 UPDATE 事件拆分成为 DELETE 和 INSERT 事件，无法正确过滤
```
Comment (asddongmen) [inline ticdc/ticdc-changefeed-config.md:107]:
```suggestion
# ignore-event = ["drop table", "delete"] # 忽略 DROP TABLE 的 DDL 事件以及 DELETE 类型的 DML 事件。
需要注意的是，在 TiDB 中，当更新聚簇索引的列值时，会将一个 UPDATE 事件拆分为 DELETE 和 INSERT 两个事件。因此，无法通过简单的过滤条件来正确忽略所有与更新聚簇索引列值相关的操作。
```

File: ticdc/ticdc-changefeed-config.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ rules = ['*.*', '!test.*']
 # 第二个事件过滤器规则
 # [[filter.event-filters]]
 # matcher = ["test.fruit"] # 该事件过滤器只应用于 test.fruit 表
-# ignore-event = ["drop table", "delete"] # 忽略 drop table 的 DDL 事件和 delete 类型的 DML 事件
+# ignore-event = ["drop table", "delete"] # 忽略 drop table 的 DDL 事件和 delete 类型的 DML 事件。需要注意的是，在 TiDB 中，当更新聚簇索引的列值时，会将一个 UPDATE 事件拆分为 DELETE 和 INSERT 两个事件。因此，无法通过简单的过滤条件来正确忽略所有与更新聚簇索引列值相关的操作。
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:107]:
```suggestion
# ignore-event = ["drop table", "delete"] # 忽略 drop table 的 DDL 事件和 delete 类型的 DML 事件。需要注意的是，TiDB 在更新聚簇索引的列值时，会将一个 UPDATE 事件拆分成为 DELETE 和 INSERT 事件，TiCDC 无法将该类事件识别为 UPDATE 事件，因此无法正确地进行过滤。
```

Comment (qiancai) [reply ticdc/ticdc-changefeed-config.md:107]: 看来起来可以参考 Event filter 时间过滤器这里的说法
https://docs.pingcap.com/zh/tidb/stable/ticdc-filter#event-filter-%E4%BA%8B%E4%BB%B6%E8%BF%87%E6%BB%A4%E5%99%A8-%E4%BB%8E-v620-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5

image

**Review by asddongmen — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**